pages/sofortige-hilfe.tsx

```diff
@@ -43,6 +43,7 @@ const Home: NextPage = () => {
         ) as keyof typeof texts
       ]
 
+  console.log(query)
```

Comment:
That log was hiding well. Probably there is no reason to keep it, right?

Reply:
Good catch!